witnet.toml

```diff
@@ -114,9 +114,9 @@ known_peers = [
 outbound_limit = 8
 # Period for opening new peer connections while the current number of peers is lower than `outbound_limit`.
 bootstrap_peers_period_seconds = 1
-# Reject (tarpit) inbound connections coming from addresses in the same /16 IP range, so as to prevent sybil peers from
+# Reject (tarpit) inbound connections coming from addresses in the same /x IP range, so as to prevent sybil peers from
 # monopolizing our inbound capacity (128 by default).
-reject_sybil_inbounds = true
+reject_sybil_inbounds = 14
```

Comment:
This change breaks existing `witnet.toml` files, and requires manual intervention to fix.

I propose to keep `true` as a legacy option and alias it to `16`.

Reply:
Makes sense

Reply:
Maybe change the name of the param is better :)

Reply:
Create two params? `reject_sybil_inbounds` set to `true` or `false` and `reject_sybil_inbounds_bits` set to an integer value? That would be a non-breaking change (I guess), and IMHO be more clear?

Reply:
yep! exactly that! :clap: